zerver/webhooks/lidarr/doc.md

```diff
@@ -0,0 +1,24 @@
+Receive Lidarr notifications in Zulip!
+
+1.  {!create-stream.md!}
+
+1.  {!create-bot-construct-url-indented.md!}
+
+1.  Go to your Lidarr
+    dashboard. Click **Settings** and then **Connect**.
+    Click the **+** icon.
+
+1.  Select **Webhook**. Enter **Zulip** or any other name you wish in the
+    name field. Tick which scenarios you would like to be notified for. You
+    can enter tags if you would only like to notify for artists with a
+    specific tag.
+
+1.  Enter the **URL** constructed earlier in the **URL** field. Ensure the
+    **Method** field is set to **POST**. Ensure the **Username** and
+    **Password** field are empty.
```

Comment:
Slight variation:

Set **URL** to the URL constructed above. Set **Method** to **POST** and leave the **Username** and **Password** fields blank.